package.js

```diff
@@ -1,22 +1,26 @@
 Package.describe({
   summary: 'Style with attitude. Sass and SCSS support for Meteor.js.',
-  version: '4.15.1',
+  version: '4.16.0',
   git: 'https://github.com/Meteor-Community-Packages/meteor-scss.git',
   name: 'fourseven:scss',
 });
 
 Package.registerBuildPlugin({
   name: 'compileScssBatch',
-  use: ['caching-compiler@1.2.2 || 2.0.0', 'ecmascript@0.15.1'],
+  use: [
+    'caching-compiler',
+    'ecmascript'
+  ],
```

Comment:
I tried just removing it but it complained when I tried to publish, this way worked:

<img width="734" alt="image" src="https://github.com/Meteor-Community-Packages/meteor-scss/assets/17411182/540cbd7a-d657-4062-a3b0-7722cfebee20">